### PR TITLE
Added some contribution guidelines.

### PR DESCRIPTION
**Pull request by toddmoy**
Started roughing in some contribution guidelines based on questions I had while submitting my first PR. I mostly assumed this was our flow, so any changes / suggestions / edits are welcomed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,6 +19,33 @@ Our Design System is designed in Sketch and [stored in Abstract](https://share.g
 
 The most obvious candidate are those design patterns (or similar variants) used in multiple products. If you have a component you think we should be using or think that we could be doing something better, these are also good bets. Not sure? Ask in [#proj-product-ds](https://hashicorp.slack.com/messages/C7KTUHNUS/)
```
Comment (joshuaogle) [inline README.md]: Mind if we fix a typo while we're in here?

```suggestion
The most obvious candidates are those design patterns (or similar variants) used in multiple products. If you have a component you think we should be using or think that we could be doing something better, these are also good bets. Not sure? Ask in [#proj-product-ds](https://hashicorp.slack.com/messages/C7KTUHNUS/)
```

**Review by joshuaogle — APPROVED** [review]
👍 Great job breaking down what I started, I had planned to go through it to see what you all think. 

Glad you figured out the Google Drive solution too, that seems to be the way to go for now but the permissions part is a pain.

**Comment by toddmoy** [comment]
Good catch! 

Also, is there a place you can directly access the image URL? I couldn't find it, so I assembled the links using copy / paste. 

If not, it might be possible to create a bookmarklet that does that work.

**Comment by joshuaogle** [comment]
Nope, the link it copies doesn't work in an image tag. No idea why they did it that way, but we'll have to change it

**Comment by toddmoy** [comment]
Cool. I made a [super janky bookmarklet](https://gist.github.com/toddmoy/3277634198876016c73dda3206d2d41d) in the meantime. Not my finest work, but it makes that process a little more manageable. If I get some time, I'll try to improve it.

**Pull request merged**